styles/main.css

```
@@ -0,0 +1,56 @@
+.main-title-box {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-bottom: 1.25rem;
+}
+
+.main-title {
+  margin-bottom: 0.9375rem;
+}
+
+.search-container {
+  height: 2.5rem;
+  padding: 0.5625rem 1.25rem;
+  border: 1px solid #dedede;
+  display: flex;
+  width: 100%;
+}
```

```suggestion
.search-container {
  display: flex;
  width: 100%;
  height: 2.5rem;
  
  padding: 0.5625rem 1.25rem;
  
  border: 1px solid #dedede;
}
```

저번에 지수님한테는 리뷰 안 드렸었나요..?!
이런 식으로 비슷한 역할끼리 묶어서 개행을 섞어주면 가독성이 좋아집니다!

---

적용한 부분도 있었는데 몇개 뺴먹었네요ㅜ 수정했습니다!
[Style: CSS 가독성 개선](https://github.com/KDT1-FE/Y_FE_JAVASCRIPT_PICTURE/pull/11/commits/89f6414cd114228ee35414b705e105a79b26bf78)